examples/loader-google-bigquery/src/data/google-bigquery.js

```diff
@@ -0,0 +1,20 @@
+import "dotenv/config";
+import {BigQuery} from "@google-cloud/bigquery";
+
+const {BQ_PROJECT_ID, BQ_CLIENT_EMAIL, BQ_PRIVATE_KEY} = process.env;
+
+if (!BQ_PROJECT_ID) throw new Error(".env missing BQ_PROJECT_ID");
+if (!BQ_CLIENT_EMAIL) throw new Error(".env missing BQ_CLIENT_EMAIL");
+if (!BQ_PRIVATE_KEY) throw new Error(".env missing BQ_PRIVATE_KEY");
+
+const bigQueryClient = new BigQuery({
+  projectId: BQ_PROJECT_ID,
+  credentials: {
+    client_email: BQ_CLIENT_EMAIL,
+    private_key: BQ_PRIVATE_KEY.replace(/\\n/g, "\n")
```

Comment:
```suggestion
    private_key: BQ_PRIVATE_KEY
```

We shouldn’t do this here. (See the google-analytics example for precedent.) If you want to use escape sequences in your environment variables in your `.env`, you need to put double quotes around the value.